patches/openwrt/0006-lantiq-xway-remove-redundant-wifi-led-on-fritz-box-7320.patch

```diff
@@ -0,0 +1,29 @@
+From 509894cffb7ec24ac21a381bed3c035efce42aeb Mon Sep 17 00:00:00 2001
+From: Manuel Kock <github.web@manu.li>
+Date: Mon, 11 Nov 2019 13:01:33 +0100
+Subject: [PATCH] lantiq: remove redundant WiFi LED on FRITZ!Box 7320
```

Comment:
I'll pick this patch upstream so we can get rid of this patch. Will ping you when done.

Reply:
Already saw the modules bump and removed the patch ;)